arangodb-net-standard/ArangoDBErrors.cs

```diff
@@ -1082,6 +1192,14 @@ public enum ArangoDBErrors
         /// <summary>
         /// the _from or _to collection specified for the edge refers to a vertex collection which is not used in any edge definition of the graph.
         /// </summary>
+        /// <remarks>
+        /// Renamed to <see cref="ERROR_GRAPH_REFERENCED_VERTEX_COLLECTION_NOT_PART_OF_THE_GRAPH"/> in ArangoDB 3.12, value not changed (1947).
+        /// </remarks>
+        [Obsolete]
```

Comment:
❓ Maybe we don't need to use `[Obsolete]`? As the value as not changed, might be enough to mention that it was renamed.

Reply:
I think it's useful leaving both, and prevents a breaking change if someone already uses this to refer to that value I think?

Reply:
Ok, I'll remove the `[Obsolete]` attribute 👍